src/main/java/de/tum/in/www1/bamboo/server/ServerNotificationRecipient.java

```diff
@@ -1,82 +1,80 @@
 import com.atlassian.bamboo.resultsummary.ResultsSummary;
-import com.atlassian.bamboo.storage.StorageLocationService;
 import com.atlassian.bamboo.template.TemplateRenderer;
 import com.atlassian.bamboo.variable.CustomVariableContext;
 import com.google.common.collect.Lists;
 import com.google.common.collect.Maps;
-import org.apache.commons.lang3.StringUtils;
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
 
-import java.util.List;
-import java.util.Map;
-import java.util.concurrent.ConcurrentHashMap;
-
-public class ServerNotificationRecipient extends AbstractNotificationRecipient implements DeploymentResultAwareNotificationRecipient,
-                                                                                           NotificationRecipient.RequiresPlan,
-                                                                                           NotificationRecipient.RequiresResultSummary
+public class ServerNotificationRecipient extends AbstractNotificationRecipient
+        implements DeploymentResultAwareNotificationRecipient, NotificationRecipient.RequiresPlan, NotificationRecipient.RequiresResultSummary
 
 {
+
     private static String WEBHOOK_URL = "webhookUrl";
 
     private String webhookUrl = null;
 
     private TemplateRenderer templateRenderer;
 
     private ImmutablePlan plan;
+
     private ResultsSummary resultsSummary;
+
     private DeploymentResult deploymentResult;
+
     private CustomVariableContext customVariableContext;
+
     private static BuildLoggerManager buildLoggerManager;
+
     private static BuildLogFileAccessorFactory buildLogFileAccessorFactory;
 
     // Time in seconds before removing ResultsContainer
     private static final int TESTRESULTSCONTAINER_REMOVE_TIME = 60;
 
     /*
-     * The TestResults of successful tests can not be loaded from the ChainResultSummary (only failing ones can).
-     * The TestResults need a BuildContext and can therefor only be accessed when using an Event that extends BuildContextEvent.
-     * The normal BuildCompletedEvent does not extend BuildContextEvent, but the PostBuildCompletedEvent does.
-     * We listen for the PostBuildCompletedEvent and save the test cases in this Map with the key of the job as String.
-     * The TestResults are stored inside the ResultsContainer class and it can be retrieved using this Map in the ServerNotificationTransport class.
-     * A method clearOldTestResultsContainer() has been added, that removes old ResultsContainer from the Map, because we add every build to this Map, even those without
-     * Notifications enabled.
-     * The time (in seconds) after the ResultsContainer can be specified in the variable TESTRESULTSCONTAINER_REMOVE_TIME.
+     * The TestResults of successful tests can not be loaded from the ChainResultSummary (only failing ones can). The TestResults need a BuildContext and can therefor only be
```

Comment:
I actually preferred the old formatting here. Can we disable the formatter for this part?

Reply:
I have to look into the options file.

Reply:
Deactivated the option for formatting of block comments and returned to the old state.